tests/test_models.py

```diff
@@ -65,15 +65,15 @@ def tearDown(self):
     #  T E S T   C A S E S
     ######################################################################
 
-    def test_example_replace_this(self):
-        """It should create a Order"""
-        # Todo: Remove this test case example
-        resource = OrderFactory()
-        resource.create()
-        self.assertIsNotNone(resource.id)
-        found = Order.all()
-        self.assertEqual(len(found), 1)
-        data = Order.find(resource.id)
-        self.assertEqual(data.name, resource.name)
+    # def test_example_replace_this(self):
```

Comment:
Do we need this test case at all?

Reply:
Since you have added test_item.py and test_model.py in PR #35, we can discard test_models.py now

Reply:
Ah yes, you are right. Completely slipped my mind. Shall I do it in the other one or let @kiara0-0 remove it in this one?

Reply:
It's up to you two~

Reply:
Okay, I addressed this in my PR.. so no need to do it here then :)